websites/A/AnimesWatch/iframe.ts

```diff
@@ -0,0 +1,13 @@
+const videoFrame = new iFrame();
+
+videoFrame.on("UpdateData", async () => {
+	const video: HTMLVideoElement = document.querySelector("video");
+
+	if (video) {
+		videoFrame.send({
+			paused: video.paused,
+			duration: video.duration,
+			currentTime: video.currentTime,
```

Comment:
Is there no possibility any of these will be null? I know for other services that use iframe they add optional chaining and a `isNaN` check

Reply:
This was not resolved

Reply:
fixed